web/src/css/custom.css

```diff
@@ -152,3 +152,10 @@
   height: 100%;
   border: 0;
 }
+
+/***** HowItWorks *****/
+.featureTitle {
+  line-height: 1.73;
+  box-decoration-break: clone;
+  -webkit-box-decoration-break: clone;
+}
```

Comment:
Since we use tailwind, could we move some of this stuff to tailwind? For example it has [leading](https://tailwindcss.com/docs/line-height) class for line height. `leading-relaxed` is 1.625, maybe that is a good value? How did you pick exactly 1.73?

There is also [box-decoration-clone](https://tailwindcss.com/docs/box-decoration-break)!

So yeah, let's use those two and let's put them directly into the JSX, next to other classes.

Reply:
1.625 would cause them to merge, I test by testing multiple values -- 1.73 was the point where the two boxes do NOT overlap. 

I can add "box-decoration-clone" and remove it from this css, but I think you should leave the line-height otherwise it will still look poor using 1.625...

Reply:
There is also a way to set artibtrary values in Tailwind, with leading-[1.73], we could do that?

Reply:
@Martinsos - I've updated the PR to use only Tailwind properties and removed the changes from the custom.css.